src/renderers/webgl/WebGLTextures.js

```diff
@@ -5,6 +5,8 @@
 import { LinearFilter, LinearMipmapLinearFilter, LinearMipmapNearestFilter, NearestFilter, NearestMipmapLinearFilter, NearestMipmapNearestFilter, RGBFormat, RGBAFormat, DepthFormat, DepthStencilFormat, UnsignedShortType, UnsignedIntType, UnsignedInt248Type, FloatType, HalfFloatType, MirroredRepeatWrapping, ClampToEdgeWrapping, RepeatWrapping } from '../../constants.js';
 import { _Math } from '../../math/Math.js';
 
+var useOffscreenCanvas = null;
```

Comment:
I think there is no need to declare this variable  in module scope. `WebGLTextures` is just created ones per renderer.

Reply:
Thanks for the info, wasn't sure about it. I have amended the PR!